src/lib/components/events/event-item.svelte

```diff
@@ -29,7 +29,7 @@
     <summary class="event-body">
       <div class="event-name">
         <h2 class="headers">
-          <a href="#{info.slug}">
+          <a href="#{info.slug}" on:click={() => (details.open = !details.open)}>
```

Comment:
Just remove the `<a>` tag entirely. This JS isn't useful and introduces different behaviors with and without JS.

Reply:
Got it. Removed in <https://github.com/EthanThatOneKid/acmcsuf.com/commit/219dbbd436a8ca7433a1313a003ff2795d7f2879>.